iked/dh.c

```diff
@@ -92,6 +92,19 @@ struct kemsx_key {
 	uint8_t		initiator;
 };
 
+/* MLKEM768 with X25519 */
+int     kemsx_init1(struct dh_group *);
```

Comment:
this should be kemmx, not kemsx, s = strntrup, x = 25519, m=mlkem

Reply:
Agreed.